src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionOverviewViewController.swift

```diff
@@ -299,3 +284,30 @@ extension ExposureSubmissionOverviewViewController {
 		case imageCard = "imageCardCell"
 	}
 }
+
+// MARK: - ExposureSubmissionOverviewViewModel.
+
+struct ExposureSubmissionOverviewViewModel {
+
+	/// Sanitize the input string and assert that:
+	/// - is not empty
+	/// - length is smaller than 150 characters
```

Comment:
Typo? Should this be 80?
And will it ever be != 80?

Reply:
good catch! I rewrote the comment, in the new implementation we guarantee that the resulting guid has a length of exactly 43 ( = 6 chars for a random number + 32 chars for the uuid + 5 chars as separators) so we can be even more narrow here 💯